samples/media/video/src/main/java/com/example/platform/media/video/TransformerVideoComposition.kt

```diff
@@ -281,13 +311,16 @@ class TransformerVideoComposition : Fragment() {
          * Class Tag
          */
         private val TAG = TransformerVideoComposition::class.java.simpleName
+
         /**
          * Video and audio assets
          */
-        private const val URI_ITEM1 =
+        private const val URI_VIDEO1 =
             "https://storage.googleapis.com/exoplayer-test-media-1/mp4/android-screens-10s.mp4"
-        private const val URI_ITEM2 =
+        private const val URI_VIDEO2 =
             "https://storage.googleapis.com/exoplayer-test-media-0/android-block-1080-hevc.mp4"
+        private const val URI_IMAGE =
+            "https://developer.android.com/static/images/media/overview/migrate-to-media3_1440.png"
```

Comment:
Using the following images resulted in an error, and I'm not sure why:
https://storage.googleapis.com/exoplayer-test-media-1/jpg/london.jpg
https://storage.googleapis.com/exoplayer-test-media-1/jpg/tokyo.jpg


```
androidx.media3.transformer.ExportException: Video frame processing error
                                                                                                    	at androidx.media3.transformer.VideoSampleExporter$VideoGraphWrapper.onError(VideoSampleExporter.java:533)
                                                                                                    	at androidx.media3.effect.SingleInputVideoGraph$1.lambda$onError$2$androidx-media3-effect-SingleInputVideoGraph$1(SingleInputVideoGraph.java:143)
                                                                                                    	at androidx.media3.effect.SingleInputVideoGraph$1$$ExternalSyntheticLambda0.run(Unknown Source:4)
                                                                                                    	at com.google.common.util.concurrent.DirectExecutor.execute(DirectExecutor.java:31)
                                                                                                    	at androidx.media3.effect.SingleInputVideoGraph$1.onError(SingleInputVideoGraph.java:143)
                                                                                                    	at androidx.media3.effect.DefaultVideoFrameProcessor$Factory$$ExternalSyntheticLambda0.onError(Unknown Source:2)
                                                                                                    	at androidx.media3.effect.VideoFrameProcessingTaskExecutor.release(VideoFrameProcessingTaskExecutor.java:179)
```

Error doesn't seem like `.jpg` specific because https://upload.wikimedia.org/wikipedia/commons/e/e6/Android_vector.jpg works fine.

Reply:
what device are you testing on?

Reply:
I tested in the demo app and it works fine with tokyo and london image. I will check out this PR and investigate

Reply:
Tested with both images: tokyo and london and it is working fine

Reply:
Testing on Pixel 6, Android 14

Reply:
Retested and can confirm that this works on my Pixel 6.

However, also realized that I was testing on an emulator (Pixel 6, API 33) when this was failing. The tokyo and london images still fail on the emulator.